Packs/FeedCyrenThreatInDepth/Scripts/CyrenThreatInDepthRenderRelated/CyrenThreatInDepthRenderRelated.py

```diff
@@ -0,0 +1,124 @@
+from CommonServerUserPython import *
+
+from enum import Enum
+from datetime import datetime
+import json
+
+
+SCORE_TO_REPUTATION_TEXT = {
+    Common.DBotScore.NONE: f"None ({Common.DBotScore.NONE})",
+    Common.DBotScore.GOOD: f"Good ({Common.DBotScore.GOOD})",
+    Common.DBotScore.SUSPICIOUS: f"Suspicious ({Common.DBotScore.SUSPICIOUS})",
+    Common.DBotScore.BAD: f"Bad ({Common.DBotScore.BAD})",
+}
+
+
+class AcceptedHeader(str, Enum):
+    INDICATOR_TYPE = "Indicator Type"
+    VALUE = "Value"
+    REPUTATION = "Reputation"
+    RELATIONSHIP_TYPE = "Relationship Type"
+    ENTITY_CATEGORY = "Entity Category"
+    TIMESTAMP = "Timestamp UTC"
+
+
+ACCEPTED_HEADERS = [
+    AcceptedHeader.INDICATOR_TYPE, AcceptedHeader.VALUE,
+    AcceptedHeader.REPUTATION, AcceptedHeader.RELATIONSHIP_TYPE,
+    AcceptedHeader.ENTITY_CATEGORY, AcceptedHeader.TIMESTAMP
+]
+
+
+def check_acceptable_headers(headers):
+    for header in headers:
+        if header not in ACCEPTED_HEADERS:
+            raise ValueError(f"Please provide columns from {ACCEPTED_HEADERS}!")
+
+
+def create_relationship_object(value: str, relationship_type: str, indicator_type: str,
+                               timestamp: str, entity_category: str, reputation: int = Common.DBotScore.NONE):
+    try:
+        timestamp_parsed = datetime.fromisoformat(timestamp.replace("Z", "+00:00"))
+        timestamp_human_readable = timestamp_parsed.strftime("%Y-%m-%d, %H:%M:%S")
+    except ValueError:
+        demisto.info("could not parse timestamp %s, keeping original string", timestamp)
+        timestamp_human_readable = timestamp
+
+    indicator_object = {
+        "Value": f"{value}\n",
+        "Reputation": SCORE_TO_REPUTATION_TEXT[reputation],
+        "Relationship Type": relationship_type,
+        "Indicator Type": indicator_type,
+        "Timestamp UTC": timestamp_human_readable,
+        "Entity Category": entity_category,
+    }
+
+    return indicator_object
+
+
+def cyren_feed_relationship(args) -> CommandResults:
+    if "indicator" not in args:
+        raise ValueError("Please provide 'indicator' argument!")
+
+    indicator = args["indicator"]
+    if isinstance(indicator, str):
+        try:
+            indicator = json.loads(indicator)
+        except json.JSONDecodeError:
+            raise ValueError("Please provide JSON-encoded 'indicator' param!")
+    elif not isinstance(indicator, dict):
+        raise ValueError("Please provide JSON-encoded 'indicator' param!")
+
+    headers = args.get("columns", ACCEPTED_HEADERS)
+    if isinstance(headers, str):
+        headers = [s.strip() for s in headers.split(",")]
+
+    check_acceptable_headers(headers)
+
+    relationships = indicator.get("CustomFields", {}).get("cyrenfeedrelationships", []) or []
+
+    content = []
+
+    for item in relationships:
+        ioc_value = item.get("value", "")
+        results = demisto.searchIndicators(value=ioc_value).get("iocs", [])
+
+        if results:
```

Comment:
```suggestion
        if results and isinstance(results, list):
```

Will it always be a list? can't is be one item?

Reply:
This comes out of the common Python functionality and I researched other usages. This seems to be guaranteed.